src/osd/modules/debugger/win/consolewininfo.cpp

```diff
@@ -112,6 +295,10 @@ void consolewin_info::set_cpu(device_t &device)
 }
 
 
+//-------------------------------------------------
+//  recompute_children
```

Comment:
Same with this, and the rest of them.  This increases line count for no useful purpose.